docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -387,15 +464,26 @@ kotlin.build.report.single_file=some_filename
 # Optional. Output directory for file-based reports. Default: build/reports/kotlin-build/
 kotlin.build.report.file.output_dir=kotlin-reports
 
-# Mandatory if HTTP output is used. Where to post HTTP(S)-based reports
+# Optional. Label for marking your build report (for example, debug parameters)
+kotlin.build.report.label=some_label
+```
+
+Options, applicable only to HTTP:
+
+```none
+# Mandatory. Where to post HTTP(S)-based reports
 kotlin.build.report.http.url=http://127.0.0.1:8080
 
 # Optional. User and password if the HTTP endpoint requires authentication
 kotlin.build.report.http.user=someUser
 kotlin.build.report.http.password=somePassword
 
-# Optional. Label for marking your build report (for example, debug parameters)
-kotlin.build.report.label=some_label
+# Optional. Add a Git branch name of a build to a build report
+kotlin.build.report.http.include_git_branch.name=true|false
+
+# Optional. Add compiler arguments to a build report
+# If a project contains many modules, its compiler arguments in the report can be very heavy and pretty useless
```

Comment:
Just wondering what you mean by pretty useless? Do they make the report too verbose?

Reply:
😄 Then maybe we could say "and not that helpful" instead? "Pretty useless" is a bit too informal and sounds very negative.